src/components/MyTimeComponents/MyTimeline.tsx

```diff
@@ -17,15 +17,17 @@ export const MyTimeline = ({ myTasks }: {myTasks: GoalItem[]}) => {
     <div id="MTL-times">
       {vbarUp && <div className="bar" />}
       {tasks.map((task: GoalItem) => {
-        const time = task.start?.toLocaleTimeString();
+        const hrs = task.start?.getHours() || 0;
+        const mins = task.start?.getMinutes() || 0;
         return (
           <>
             <button
               type="button"
               onClick={() => handleDisplayOptions(task)}
               className="MTL-startTime"
             >
-              {time?.slice(0, -6)} {time?.slice(-2)}
+              {`${Math.floor(hrs / 10) === 0 ? 0 : ""}${hrs}:`}
+              {`${Math.floor(mins / 10) === 0 ? 0 : ""}${mins}`}
```

Comment:
Why are we doing custom logic here?
Date.getHours() and Date.getMinutes() should already give the required info.

Reply:
@tijlleenders Actually, it's for keeping the hrs and minutes in 2 digits. For e.g., if the time 05:15 then without this logic the getHours() will give us 5:15 so to keep things similar I wrote this logic. In-built function returns number

Reply:
```suggestion
              {task.start?.toTimeString().slice(0, 5)};
```

Reply:
This works and is easier to read the intent from.
Definition of hrs and mins earlier in code can then also be deleted.